From 425acff1f944fa7526942a9fd2d61b74d26b0a22 Mon Sep 17 00:00:00 2001
From: Anthony Rabbito <hello@anthonyrabbito.com>
Date: Sat, 4 Sep 2021 23:52:28 -0400
Subject: [PATCH 1/2] Squashed commit of the following:

commit b053ad7d4e88c54473b667284577f88156b04841
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 23:16:58 2021 -0400

    fix: syntax

    Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>

commit c057a533d1c65949bf7b7ce14b02777bcde0fdfe
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 22:28:23 2021 -0400

    fix: add oars

    Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>

commit eb10a8896c26b9466bf96ce87cccdc2f06ee0aed
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 22:27:50 2021 -0400

    Revert "fix: appstream metadata"

    This reverts commit ee63b7381c25a55fe2f27d3cbb457e01ae813e9a.

commit ee63b7381c25a55fe2f27d3cbb457e01ae813e9a
Author: Anthony Rabbito <ted437@gmail.com>
Date:   Fri Sep 3 22:33:01 2021 -0400

    fix: appstream metadata

    * Add oars rating
    * Add release

    Signed-off-by: Anthony Rabbito <ted437@gmail.com>

Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>
---
 org.kde.artikulate.appdata.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/org.kde.artikulate.appdata.xml b/org.kde.artikulate.appdata.xml
index 6403f4c..12729d4 100644
--- a/org.kde.artikulate.appdata.xml
+++ b/org.kde.artikulate.appdata.xml
@@ -164,4 +164,5 @@ SPDX-License-Identifier: CC0-1.0
   <provides>
     <binary>artikulate</binary>
   </provides>
+  <content_rating type="oars-1.1"/>
 </component>
-- 
2.32.0


From e2e39e9b20e1e9681db639c2c931e289ac45a98a Mon Sep 17 00:00:00 2001
From: Anthony Rabbito <hello@anthonyrabbito.com>
Date: Sat, 4 Sep 2021 23:53:55 -0400
Subject: [PATCH 2/2] Squashed commit of the following:

commit 0f0dfa4fb5ff9a1ad9c720205e4ef30b3226aaa0
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 23:23:15 2021 -0400

    fix: add release to appstream

    Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>

commit c707748583b73334ae03d57b7762d956909e804b
Author: Anthony Rabbito <hello@anthonyrabbito.com>
Date:   Sat Sep 4 23:10:17 2021 -0400

    fix: use non-gear versioning

    Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>

Signed-off-by: Anthony Rabbito <hello@anthonyrabbito.com>
---
 CMakeLists.txt                 | 11 +++++++++--
 org.kde.artikulate.appdata.xml |  3 +++
 src/CMakeLists.txt             |  2 +-
 3 files changed, 13 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ad049f..a3defe3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,12 +1,19 @@
 # SPDX-FileCopyrightText: 2013-2016 Andreas Cord-Landwehr <cordlandwehr@kde.org>
 # SPDX-License-Identifier: BSD-2-Clause
 
-project(artikulate)
-
 cmake_minimum_required(VERSION 3.5.0)
 set(QT_MIN_VERSION "5.11.0")
 set(KF5_MIN_VERSION "5.64.0")
 
+# KDE Application Version, managed by release script
+set (RELEASE_SERVICE_VERSION_MAJOR "21")
+set (RELEASE_SERVICE_VERSION_MINOR "11")
+set (RELEASE_SERVICE_VERSION_MICRO "70")
+set (RELEASE_SERVICE_COMPACT_VERSION "${RELEASE_SERVICE_VERSION_MAJOR}${RELEASE_SERVICE_VERSION_MINOR}${RELEASE_SERVICE_VERSION_MICRO}")
+set (ARTIKULATE_VERSION "1.0.0.${RELEASE_SERVICE_COMPACT_VERSION}")
+
+project(artikulate VERSION ${ARTIKULATE_VERSION})
+
 find_package(ECM ${KF5_MIN_VERSION} REQUIRED NO_MODULE)
 find_package(KF5DocTools)
 set(CMAKE_MODULE_PATH ${ECM_MODULE_PATH} )
diff --git a/org.kde.artikulate.appdata.xml b/org.kde.artikulate.appdata.xml
index 12729d4..16003e5 100644
--- a/org.kde.artikulate.appdata.xml
+++ b/org.kde.artikulate.appdata.xml
@@ -165,4 +165,7 @@ SPDX-License-Identifier: CC0-1.0
     <binary>artikulate</binary>
   </provides>
   <content_rating type="oars-1.1"/>
+  <releases>
+    <release version="1.0.0.211170" date="2021-09-04"/>
+  </releases>
 </component>
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c3d6d61..7ab3718 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: 2013-2019 Andreas Cord-Landwehr <cordlandwehr@kde.org>
 # SPDX-License-Identifier: BSD-2-Clause
 
-ecm_setup_version(1.0.0
+ecm_setup_version(${ARTIKULATE_VERSION}
     VARIABLE_PREFIX ARTIKULATE
     VERSION_HEADER "${CMAKE_CURRENT_BINARY_DIR}/version.h"
     PACKAGE_VERSION_FILE "${CMAKE_CURRENT_BINARY_DIR}/ArtikulateConfigVersion.cmake"
-- 
2.32.0

